src/components/Authentication/Kratos/messages.ts

```diff
@@ -0,0 +1,36 @@
+import { UiText } from '@ory/kratos-client/api';
+import { has } from 'lodash';
+
+/**
+ * A message from Kratos is received in the form of an object with an id property. Kratos docs say:
+ * This ID will not change and can be used to translate the message or use your own message content.
+ * See https://www.ory.sh/docs/kratos/concepts/ui-user-interface
+ */
+
+const messages = {
+  '4070005': 'verification-flow-expired',
+};
+
+export default messages;
+
+interface T {
+  (
+    path: string,
+    replacement?:
+      | {
+          [key: string]: string | number;
+        }
+      | undefined
+  ): string;
+}
+
+export const KratosFriendlierMessageMapper =
+  (t: T) =>
```

Comment:
you can import `getI18n`  and use it directly without the need to be provided here.